governance-exercises/poll-ex1-response.md

```diff
@@ -0,0 +1,46 @@
+# StableLab Polling Exercise Responses 
+
+## 1: Determine what course of action you would take as a Governance Facilitator and communicate it clearly
+
+- I would first review the proposal thoroughly and perform any necessary compliance checks
+- I would then reach out to Alice and Faraz if any outstanding details or any ambiguities require further clarification which would determine what polls should be drafted. In this case, the proposal provides all details required.
+- Based on Maker's Governance cycles as detailed [here](https://manual.makerdao.com/governance/governance-cycle), I would assess the suitability of the proposal's polls for either the Weekly or Monthly Governance cycle. Since the proposal author has emphasized the time-sensitivity of the matter, citing their user base's rapid growth trajectory and the need to capitalize on this growth, I would opt to include these polls in the Weekly cycle.
+- Since Bob has already given authorization via forum post for these Spork-related polls, I would then determine how to split these polls so that users are allowed to express their opinions separately.
+- While these actions are closely linked, I would split them into two separate polls to clearly distinguish between the distinct proposed changes.
+  
+  ### Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023:
+  - Increase Spork D3M Debt Ceiling by 50 million from 50 million to 100 million DAI.
+  - Increase Spork D3M Target Available Debt (gap) by 10 million from 5 million to 15 million.
+  - Adjust Spork Protocol DAI Interest Rate to increase the DAI deposit rate by 0.5% from 2% to 2.5%
+  - Increase the Ceiling Increase Cooldown from 28,800 seconds (8 hrs) to 86,400 seconds (24 hrs)
+ 
+  ### Onboard ABC to Spork Protocol - July 24, 2023
+  -  Onboard the token ABC to Spork with the parameters below:
+	- [onboarding parameters for token $ABC]
+
+- After determining the contents of the poll I would choose from the templates folder the most suitable templates to slot this poll content into, which in this case are 'Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md' and 'Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md'.
+- Next, I would publish a response in the forum thread informing the proposers and community of the decision to submit two polls, with rationale included. *See forum response below*
+- I would then begin to draft the polls using the poll templates as noted above. The polls will include a summary of the proposal, voting choices, and links to relevant resources for voters to make informed decisions. I will review the poll drafts against the 'poll creation checklist' to ensure they meet all criteria. Additionally, I will carefully check for any spelling or grammatical errors and make sure the language used is easy to understand.
+- I would then create a draft PR to the repo and request reviews of the polls content from other scope facilitators, the proposers and Bob
```

Comment:
Generally speaking, other stakeholders don't want to be reviewing the poll wording. As facilitator, part of the job is ensuring that they don't need to. In an ideal world, you already have a clear idea of what they are committing to, or, if they are not committing to anything specific, and can ensure the wording reflects that.

Reply:
I would confirm prior to posting publicly that the stakeholders are satisfied with the separation into separate polls.